test/distributed/org/apache/cassandra/distributed/test/ring/BootstrapTest.java

```diff
@@ -110,15 +112,31 @@ public void hostIdOverrideTest() throws Throwable
             populate(cluster,0, 100);
             IInstanceConfig config = cluster.newInstanceConfig();
             IInvokableInstance newInstance = cluster.bootstrap(config);
+
+            
             withProperty("cassandra.host_id_first_boot", "06fc931f-33b5-4e22-0001-000000000001",
                          () -> newInstance.startup(cluster));
 
             newInstance.shutdown().get();
```

Comment:
no need to stop instance here before doing assertion

Reply:
removed